ui/ducks/metamask/metamask.js

```diff
@@ -340,3 +341,7 @@ export function getIsUnlocked(state) {
 export function getSeedPhraseBackedUp(state) {
   return state.metamask.seedPhraseBackedUp;
 }
+
+export function getIsNetworkCongested(state) {
+  return state.metamask.isNetworkCongested;
```

Comment:
This piece of data is populated in GasFeeController. You can see it in action in this PR: https://github.com/MetaMask/controllers/pull/609